jspsych-uil-session.js

```diff
@@ -23,6 +23,7 @@ export {
     isActive,
     start,
     upload,
+    uploadBinary,
```

Comment:
As this function is public, it would be nice to get it into the documentation: /doc/lib/session.rst